elasticdl/python/common/args.py

```diff
@@ -159,6 +159,9 @@ def _check_master_args_validity(args):
         logger.warning(
             "grads_to_wait is set to 1 while using asynchronous SGD."
         )
+    if args.num_ps_pods == 0:
+        logger.warning("AllReduce strategy does not need PS")
```

Comment:
if args.num_ps_pods == 0 && args.distribution_strategy != DistributionStrategy.ALLREDUCE:
    logger.warning("Set the distribution strategy to be AllReduce if ps number is 0.")

Reply:
Yeah, fixed it.